opencog/combo/converter/combo_atomese.h

```diff
@@ -52,10 +52,11 @@ class ComboToAtomeseConverter
 	 * @param combo_tree   the combo_tree containing the combo program
 	 * @return                 the Handle containing the atomese program
 	 */
-	Handle operator()(const combo_tree &tr);
+	Handle operator()(const combo_tree &tr, std::string suffix="");
```

Comment:
If `=""` doesn't work because it is passed by const ref, you can use `=std::string()`.